libyul/backends/evm/ControlFlowGraphBuilder.cpp

```diff
@@ -51,12 +52,27 @@ namespace
 /// Removes edges to blocks that are not reachable.
 void cleanUnreachable(CFG& _cfg)
 {
+	// If operation is a function call it adds the callee entry as child
+	auto const addFunctionsEntries = [&_cfg](CFG::BasicBlock* _node, auto&& _addChild)
+	{
+		for (auto const& o: _node->operations)
+		{
+			if (auto const* p = std::get_if<CFG::FunctionCall>(&o.operation); p != nullptr)
+			{
+				auto const fInfo = _cfg.functionInfo.at(&(p->function.get()));
```

Comment:
The naming is pretty bad here. And the `nullptr` check seems redundant.

```suggestion
			if (auto const* functionCall = std::get_if<CFG::FunctionCall>(&operation.operation))
			{
				auto const functionInfo = _cfg.functionInfo.at(&(functionCall->function.get()));
```

Reply:
I definitely agree.